package.json

```diff
@@ -48,9 +48,9 @@
   "devDependencies": {
     "@babel/cli": "^7.5.5",
     "@babel/core": "^7.5.5",
-    "@babel/runtime": "^7.21.0",
     "@babel/plugin-transform-runtime": "^7.5.5",
     "@babel/preset-env": "^7.5.5",
+    "@babel/runtime": "^7.21.0",
```

Comment:
`yarn install` seems to have applied this. I guess dependencies are alphabetized in Yarn v3.